pkg/sql/sqlbase/system.go

```diff
@@ -439,3 +442,7 @@ func addSystemDatabaseToSchema(target *MetadataSchema) {
 func IsSystemConfigID(id ID) bool {
```

Comment:
Yep, by `UPDATE`s, which seem to properly care whether the update is impacting the system config span or not.